src/stats.c

```diff
@@ -216,6 +230,74 @@ void stats_get(struct stats *stats)
     }
 }
 
+static void stats_send_slow_log()
+{
+    const char *fmt = "nodes.%s.slow_query.%s";
```

Comment:
Two problems here:

1. `nodes` should be changed to `redis-node` for consistency.
2. The first `%s` is the combination of ip and port, so the dot in ip should be changed to `-`. [L155-L160](https://github.com/doyoubi/corvus/blob/50e4f96a3835db3c8bacd987a4fa85be8abcaa7c/src/stats.c#L155-L160)

Maybe we can reuse the function `stats_node_info_agg` and add the `slow_counts` field to the `bytes` struct.